pandas/tests/test_indexers.py

```diff
@@ -0,0 +1,11 @@
+# Tests aimed at pandas.core.indexers
+import numpy as np
```

Comment:
did you mean to create an entirely new file (ok, but shouldn't this be in pandas/tests/indexing/test_indexers.py?)

Reply:
I did, because we don't have a dedicated place for it.  Will move to tests/indexing/test_indexers.py